### PR TITLE
chore(deps): Update to latest npm modules

### PR DESCRIPTION
**Pull request by jesseproudman**
Updates many libraries to latest versions eliminating issues via `yarn audit` and warnings packaging.

### CODE REVIEW

**Comment by jesseproudman** [comment]
Doesn't appear that there's `ci` here but the following pass locally:

```
yarn lint
yarn jest
yarn build
```

**Comment by jesseproudman** [comment]
@zerodom30 I can not. I don't have permissions on the repo. I think @smudge maybe can?

**Comment by zerodom30** [comment]
@jesseproudman no problem I tagged him

**Review by dfranco65 — APPROVED** [review]
domain checkride lgtm

File: src/mixpanelAnalytics.test.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ describe('MixpanelAnalytics', () => {
   });
 
   describe('#trackAssignment()', () => {
+    // eslint-disable-next-line jest/no-done-callback
     it('calls window.mixpanel.track()', done => {
```
Comment (zerodom30) [inline src/mixpanelAnalytics.test.js:23]: The fix for the eslint error, just use an anonymous callback and delete the call to`done`. 
```suggestion
    it('calls window.mixpanel.track()', () => {
```

File: src/session.test.js [hunk 2]
```diff
@@ -202,6 +202,7 @@ describe('Session', () => {
     });
 
     describe('#vary()', () => {
+      // eslint-disable-next-line jest/no-done-callback, jest/expect-expect
       it('calls the correct vary function for the given split', done => {
```
Comment (zerodom30) [inline src/session.test.js:206]: There should be an `expect` assertion in this test.

**Review by zerodom30 — CHANGES_REQUESTED** [review]
The eslint errors should be resolved instead of ignored since it does provide needed improvements to the tests. 
[expect expect rule](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/expect-expect.md#enforce-assertion-to-be-made-in-a-test-body-expect-expect)
[jest/no-done-callback](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-done-callback.md)

**Comment by jesseproudman** [comment]
@zerodom30 I don't know how to fix those specs -- I've given it a shot but get more errors. Could you give them a shot? You're welcome to push to the branch.

**Comment by jesseproudman** [comment]
@zerodom30 Was able to fix the specs this morning. This should be good to go now.

**Review by zerodom30 — APPROVED** [review]
platformLGTM

**Comment by jesseproudman** [comment]
@zerodom30 Looks like we've got an old policy bot configuration that's pointing at the wrong branch?

**Review by DanC5 — APPROVED** [review]
domain lgtm

**Comment by smudge** [comment]
> isn't that because this is a fork of the original repo

I'm not sure that the issue is that the PR comes from a fork -- all our public repos are set up with a forking model. I suspect the policy bot is just misconfigured.

**Pull request merged**